src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -94,4 +95,22 @@ private function loadResources()
             unset($this->resourceFiles[$key]);
         }
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function warmUp($cacheDir)
+    {
+        if (null !== $this->options['cache_dir']) {
+            if (null !== $this->locale) {
+                $this->loadCatalogue($this->locale);
```

Comment:
Sure, I planned to open another PR to warming for all locales, otherwise, bellow how to it should be:
```php
    /**
     * {@inheritdoc}
     */
    public function warmUp($cacheDir)
    {
        $locales = array();
        foreach ($this->resourceFiles as $file) {
            list($domain, $locale, $format) = explode('.', basename($file), 3);
            $locales[$locale] = $locale;
        }
        
        foreach ($locales as $locale) {
            $this->loadCatalogue($locale);
        }
    }
```